pom.xml

```diff
@@ -255,7 +255,9 @@
                 -->
                 <module>shopping-cart</module>
                 <module>spring-resteasy</module>
+                <!-- TODO It's failing
```

Comment:
If it's failing we should add spring-petclinic to https://issues.jboss.org/browse/JDF-824